$Id$

	In uClibc, clock_gettime(2) is defined as user-space
	function in <time.h> instead of <sys/time.h>, so we
	need that as well. Candidate for sending to upstream.

--- libevent-1.3.orig/event.c	2007-02-16 00:49:06.000000000 +0000
+++ libevent-1.3/event.c	2007-02-18 14:21:29.000000000 +0000
@@ -51,6 +51,9 @@
 #include <signal.h>
 #include <string.h>
 #include <assert.h>
+#ifdef __UCLIBC__
+#include <time.h>
+#endif
 
 #include "event.h"
 #include "event-internal.h"
